packages/components/psammead-caption/src/index.jsx

```diff
@@ -19,10 +26,14 @@ const Caption = styled.figcaption`
   color: ${C_CLOUD_DARK};
   font-family: ${GEL_FF_REITH_SANS};
   ${FS_ITALIC};
-  padding: ${GEL_SPACING};
+  padding: ${GEL_SPACING} 0;
   width: 100%;
-  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
-    padding: ${GEL_SPACING} 0;
+
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: ${GEL_SPACING} ${GEL_MARGIN_BELOW_400PX};
```

Comment:
Can we be mobile first please?

I'd have this padding value as the default ^ (on line 29)

Then we can remove this media block, and add one at the end instead:

```suggestion
@media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
  padding: ${GEL_SPACING} 0;
```

...so that it reads from mobile to bigger.

Current implementation defaults to the desktop view and then overrides for specific smaller breakpoints.

Reply:
done :)